R/seqSimVis.R

```diff
@@ -0,0 +1,98 @@
+
+
+#' \code{seqSimVis()} Calculates sequence similarity and plots dot plots based on the
+#' three highest similarity values to visualize
```

Comment:
Maybe you could specify the number of top similarity values to visualize as a parameter for your function, and set 3 to be the default value.

Reply:
Again - I don't see how this was "resolved" ... :-(